force_wfmanager/central_pane/analysis.py

```diff
@@ -1,9 +1,52 @@
 from pyface.tasks.api import TraitsTaskPane
-from traitsui.api import View
+
+from traits.api import List, Instance, Int, Any, Str, HasStrictTraits
+
+from traitsui.api import View, HGroup, VGroup, Spring
+
+
+class Result(HasStrictTraits):
+    name = Str()
+    value = Any()
+
+
+class EvaluationStep(HasStrictTraits):
+    _step = 0
+
+    @classmethod
+    def init_step_count(cls):
```

Comment:
And in a next PR I want to create the Table view, and in yet another PR I want to create a plot